frontend/src/main/pages/PlayPage.js

```diff
@@ -32,20 +32,19 @@ export default function PlayPage() {
     );
   // Stryker enable all 
 
-
-  // Stryker disable all 
-  const { data: commons } =
+  // Stryker disable all
+  const { data: commonsPlus } =
     useBackend(
-      [`/api/commons?id=${commonsId}`],
+      [`/api/commons/plus?id=${commonsId}`],
       {
         method: "GET",
-        url: "/api/commons",
+        url: "/api/commons/plus",
         params: {
           id: commonsId
         }
       }
     );
-  // Stryker enable all 
+  // Stryker enable all
```

Comment:
Should be 
```suggestion
  // Stryker restore all
```

Please see: https://ucsb-cs156-s23.slack.com/archives/C050779M19P/p1685726928710669